src/main/flight/pid.h

```diff
@@ -116,6 +125,12 @@ typedef struct pidProfile_s {
     uint8_t throttle_boost_cutoff;          // Which cutoff frequency to use for throttle boost. higher cutoffs keep the boost on for shorter. Specified in hz.
     uint8_t  iterm_rotation;                // rotates iterm to translate world errors to local coordinate system
     uint8_t  smart_feedforward;             // takes only the larger of P and the D weight feed forward term if they have the same sign.
+    uint8_t iterm_relax_cutoff_low;        // Slowest setpoint response to prevent iterm accumulation
+    uint8_t iterm_relax_cutoff_high;       // Fastest setpoint response to prevent iterm accumulation
+    itermRelax_e iterm_relax;              // Enable iterm suppression during stick input
```

Comment:
Can you please change this into an `uint8_t` here? There is not much point in storing this as 32 bits in the parameter group, and this will also fix the alignment. (The rest of the code can still use the enum).

Reply:
done